### PR TITLE
Add Poli Systems

### PR DESCRIPTION
**Pull request by IIPoliII**
Please add https://polisystems.ch.

The website is ranked also below 200k -> https://www.alexa.com/siteinfo/polisystems.ch

### CODE REVIEW

**Comment by IIPoliII** [comment]
I don't understand why the first test fails directly at the start of "{"

**Comment by Carlgo11** [comment]
> Sites supporting TFA must not have a `contact` property.
> \- CONTRIBUTING.md

I think that's the reason it fails

File: entries/p/polisystems.ch.json [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+         "email":"support@polisystems.ch"
+      },
+      "documentation":"https://wiki.polisystems.ch/English/Tutorials/Two-factor-authentification-on-Poli-Systems-Portal/",
+      "keywords":[
+         "hosting",
+         "cloud",
+         "email"
+      ]
+   }
+}
```
Comment (Carlgo11) [inline entries/p/polisystems.ch.json]:
```suggestion
{
   "Poli Systems":{
      "domain": "polisystems.ch",
      "tfa": [
         "totp"
      ],
      "documentation": "https://wiki.polisystems.ch/English/Tutorials/Two-factor-authentification-on-Poli-Systems-Portal/",
      "keywords": [
         "hosting",
         "cloud",
         "email"
      ]
   }
}
```

**Review by Carlgo11 — CHANGES_REQUESTED** [review]

**Comment by IIPoliII** [comment]
@Carlgo11 Thanks for the fix didn't see that!

**Review by Carlgo11 — CHANGES_REQUESTED** [review]
The logo is unnecessarily large. Ideally it would be between <2KiB.
Try and clean it up or use this one that I've cleaned up:

**Comment by IIPoliII** [comment]
@Carlgo11 I tried to compress it a bit more than yours but I was only able to make it 0.1 Ko tiner.
It's now 1.36Ko could that work out?

File: img/p/polisystems.ch.svg [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+<stop stop-color="#ff6b89" offset="1"/>
+</linearGradient>
+<linearGradient id="b" x1="100" x2="0" xlink:href="#a"/>
+<path d="M97.33 17.84V19a15.18 15.18 0 0 1-15.17 15.18H6.26a3.57 3.57 0 0 1-3.55-3.57V6.22a3.57 3.57 0 0 1 3.57-3.57h75.86a15.18 15.18 0 0 1 15.2 15.2ZM93.42 65.7a3.93 3.93 0 0 1 3.93 3.93v23.18a4.43 4.43 0 0 1-4.28 4.45H17.89A15.18 15.18 0 0 1 2.7 82.06V50.52A15.18 15.18 0 0 0 17.89 65.7Z" fill="url(#a)"/>
+<path d="M97.3 18.99v31.53A15.18 15.18 0 0 1 82.1 65.7H17.6A15.18 15.18 0 0 1 2.65 50.52v-1.16a15.18 15.18 0 0 1 15.2-15.18h64.23a15.18 15.18 0 0 0 15.2-15.2Z" fill="url(#b)"/>
+<g stroke="none">
+<path d="M81.82 5.28a15.23 15.23 0 0 1 5.86 11.99v1.16a15.22 15.22 0 0 1-7.58 13.15h2.04c6.97 0 12.57-5.6 12.57-12.57v-1.16c0-6.97-5.6-12.57-12.58-12.57ZM18.09 36.87a15.23 15.23 0 0 0-5.86 11.99v1.16a15.22 15.22 0 0 0 7.57 13.15h-2.04c-6.97 0-12.56-5.6-12.56-12.57v-1.16c0-6.98 5.6-12.57 12.58-12.57Z" fill="#00000026"/>
+<path d="m94.86 31.66a17.84 17.84 0 0 1-7.06 4.3v15.83a15.13 15.13 0 0 1-4.92 11.17 12.45 12.45 0 0 0 11.98-12.48z" fill="#00f2ff"/>
+</g>
+</svg>
```
Comment (Carlgo11) [inline img/p/polisystems.ch.svg]:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" stroke="#000678" stroke-width="5.3" viewBox="0 0 100 100"><linearGradient id="a"><stop offset="0" stop-color="#00f2ff"/><stop offset="1" stop-color="#ff6b89"/></linearGradient><linearGradient xlink:href="#a" id="b" x1="100" x2="0" gradientUnits="userSpaceOnUse"/><path fill="url(#a)" d="M97.33 17.84V19a15.18 15.18 0 0 1-15.17 15.18H6.26a3.57 3.57 0 0 1-3.55-3.57V6.22a3.57 3.57 0 0 1 3.57-3.57h75.86a15.18 15.18 0 0 1 15.2 15.2ZM93.42 65.7a3.93 3.93 0 0 1 3.93 3.93v23.18a4.43 4.43 0 0 1-4.28 4.45H17.89A15.18 15.18 0 0 1 2.7 82.06V50.52A15.18 15.18 0 0 0 17.89 65.7Z"/><path fill="url(#b)" d="M97.3 18.99v31.53A15.18 15.18 0 0 1 82.1 65.7H17.6A15.18 15.18 0 0 1 2.65 50.52v-1.16a15.18 15.18 0 0 1 15.2-15.18h64.23a15.18 15.18 0 0 0 15.2-15.2Z"/><g stroke="none"><path fill="#00000026" d="M81.82 5.28a15.23 15.23 0 0 1 5.86 11.99v1.16a15.22 15.22 0 0 1-7.58 13.15h2.04c6.97 0 12.57-5.6 12.57-12.57v-1.16c0-6.97-5.6-12.57-12.58-12.57ZM18.09 36.87a15.23 15.23 0 0 0-5.86 11.99v1.16a15.22 15.22 0 0 0 7.57 13.15h-2.04c-6.97 0-12.56-5.6-12.56-12.57v-1.16c0-6.98 5.6-12.57 12.58-12.57Z"/><path fill="#00f2ff" d="M94.86 31.66a17.84 17.84 0 0 1-7.06 4.3v15.83a15.13 15.13 0 0 1-4.92 11.17 12.45 12.45 0 0 0 11.98-12.48z"/></g></svg>
```

**Review by Carlgo11 — APPROVED** [review]

File: entries/p/polisystems.ch.json [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+   "Poli Systems":{
+      "domain": "polisystems.ch",
+      "tfa": [
+         "totp"
+      ],
+      "documentation": "https://wiki.polisystems.ch/English/Tutorials/Two-factor-authentification-on-Poli-Systems-Portal/",
+      "keywords": [
+         "hosting",
+         "cloud",
+         "email"
```
Comment (phallobst) [inline entries/p/polisystems.ch.json]: > High-end mail hosting _included_ in your web hosting, ...

As long as they don't offer standalone mail hosting, we should not include `email` here, I think.
```suggestion
         "cloud"
```

**Review by phallobst — CHANGES_REQUESTED** [review]

**Review by phallobst — APPROVED** [review]

**Comment by Sethis11263** [comment]
Seth

**Pull request merged**